src/pages/UserSettings/Teams.vue

```diff
@@ -243,37 +243,38 @@ export default {
           "
           @decline="handleDeclinePendingInvitation(item.id, item.tenant.name)"
         />
-        <v-tooltip bottom
-          ><template #activator="{on}"
-            ><v-btn
-              v-if="item.tenant.id !== tenant.id && item.member"
-              text
-              fab
-              x-small
-              color="primary"
-              v-on="on"
-              @click="handleSwitchTenant(item)"
-              ><v-icon>fas fa-eye</v-icon></v-btn
-            ></template
-          >Switch to this team</v-tooltip
-        >
-        <v-tooltip bottom
-          ><template #activator="{on}"
-            ><v-btn
-              v-if="item.member"
-              text
-              fab
-              x-small
-              color="error"
-              v-on="on"
-              @click="
-                dialogRemoveUser = true
-                removeTenant = item.tenant
-              "
-              ><v-icon>close</v-icon></v-btn
-            ></template
-          >Leave this team</v-tooltip
-        >
+        <div style="padding-right: 6px;">
+          <v-tooltip bottom
+            ><template #activator="{on}"
+              ><v-btn
+                v-if="item.tenant.id !== tenant.id && item.member"
+                text
+                small
+                style="margin-right: 14px;"
+                color="primary"
+                v-on="on"
+                @click="handleSwitchTenant(item)"
+                ><v-icon>fas fa-eye</v-icon></v-btn
```

Comment:
this eye is so much heavier than the checkmark, I'm tempted to either shrink it or find a different icon. Anyone else feel strongly?

Reply:
I think font awesome icons tend to be larger for some reason.  You could add a 'small' prop to the v-icon?

Reply:
How about these? (`swap_horiz` and `meeting_room`)
![Screen Shot 2020-11-11 at 4 31 59 PM](https://user-images.githubusercontent.com/27291717/98866772-73d5e480-243b-11eb-917c-68252ee6dc53.png)

Reply:
I like swap_horiz, my feeling is that declining an invite and leaving a team are close enough to the same thing that they can have the same icon.